base/fontdef.dtx

```diff
@@ -411,6 +411,18 @@
 \endgroup
 %    \end{macrocode}
 %
+% \changes{v3.0j}{2024/06/24}{load ts1 cmr fd file in Unicode engines}
+% Even though Unicode engines default to |lm| load |ts1cmr| 
+% as this may be used for fallback for TS1 encoding.
```

Comment:
THinking about that, I don't think there is anything wrong to set the fallback to lm in case of Unicode engines if that font is loaded into the format. But the way it is now is probably ok as well.

Reply:
yes I did wonder but probably we say in several places that cm is the default so for now at least I just changed the preload (which shouldn't change any output as the same fd file got loaded at run time anyway) I'll merge